packages/SwingSet/src/vats/network/network.js

```diff
@@ -2,15 +2,23 @@
 import makeStore from '@agoric/store';
 import rawHarden from '@agoric/harden';
 import { E as defaultE } from '@agoric/eventual-send';
+import { producePromise } from '@agoric/produce-promise';
 import { toBytes } from './bytes';
 
 const harden = /** @type {<T>(x: T) => T} */ (rawHarden);
 
+export const ENDPOINT_SEPARATOR = '/';
```

Comment:
maybe add a comment: "compatibility note: this must match what our peers use, so don't change it casually"